test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -917,6 +917,28 @@ $powershell -c '[System.Management.Automation.Platform]::SelectProductNameForDir
             $out[0] | Should -MatchExactly $expectedPromptPattern
         }
     }
+
+    Context 'CommandWithArgs tests' {
+        It 'Should be able to run a pipeline with arguments using <param>' -TestCases @(
+            @{ param = '-commandwithargs' }
+            @{ param = '-cwa' }
+        ){
+            param($param)
+            $out = pwsh -nologo -noprofile $param '$args | % { "[$_]" }'  '$fun' '@times'
```

Comment:
Looking the example - maybe replace `-cwa` with `-cargs)?

Reply:
I think `-cwa` is shorter and also a logical/readable abbreviation to `-commandwithargs`